tools/scripts/glaux.jlink.commands

```diff
@@ -1,6 +1,6 @@
 power on perm
 exec EnableRemarks
 unlock kinetis
-loadfile <full-path-to-warp-firmware>/build/ksdk1.1/work/demos/Glaux/armgcc/Glaux/release/Glaux.srec
+loadfile /Users/hamid/warp-firmware-resolved-sleep-issue-copy/Warp-firmware/build/ksdk1.1/work/demos/Glaux/armgcc/Glaux/release/Glaux.srec
```

Comment:
Please remove this change from the PR. The change adds a variable for your local environment so it will break things for other people.